pkg/agents/team3/agentMain.go

```diff
@@ -20,12 +21,30 @@ type team3Knowledge struct {
 	floors []int
 	//We know the last HP
 	lastHP int
+	//We know who we have meet
+	//friends []string
+	//We know if we like or not the people we have met
+	//friendship []float64
+	//Stores who is in the floor bellow
+	floorBelow string
+	//Stores who is in the floor above
+	floorAbove string
+	//Stores food last eaten
+	foodLastEaten food.FoodType
+}
+
+type team3Decisions struct {
+	//Amount of food we decided to eat.
+	foodToEat int
+	//Amount of food we decided to leave in the platform.
+	foodToLeave int
```

Comment:
Ideally should be a `food.FoodType`

Reply:
Breaks too many things right now to do that but I will take it into account for the next commit